examples/05_Collections/14_sorted.md

```diff
@@ -8,17 +8,23 @@
 fun main() {
 
 //sampleStart
-    val shuffled = listOf(5, 4, 2, 1, 3)     // 1
-    val natural = shuffled.sorted()          // 2
-    val inverted = shuffled.sortedBy { -it } // 3
+    val shuffled = listOf(5, 4, 2, 1, 3)                   // 1
+    val natural = shuffled.sorted()                        // 2
+    val inverted = shuffled.sortedBy { -it }               // 3
+    val descending = shuffled.sortedDescending()           // 4
+    val descendingBy = shuffled.sortedByDescending { it  } // 5
```

Comment:
This example seems a bit unnatural since it repeats the previous one with longer code.
We can add some negative values to the original list and sort it by descending of the item's absolute value (use `Math.abs(it)` as a selector. Don't forget to change the description text.

Reply:
Thanks for the review. Done.